controller/space_codebases_test.go

```diff
@@ -130,7 +130,7 @@ func (rest *TestSpaceCodebaseREST) TestCreateCodebaseCreated() {
 	require.NotNil(t, c.Data.Relationships.Space)
 	assert.Equal(t, sp.ID.String(), *c.Data.Relationships.Space.Data.ID)
 	require.NotNil(t, c.Data.Attributes.CveScan)
-	assert.Equal(t, true, *c.Data.Attributes.CveScan)
+	assert.Equal(t, false, *c.Data.Attributes.CveScan)
```

Comment:
Please use proper method to test: `assert.False` does exist for this purpose.

Reply:
done!